aea/agent.py

```diff
@@ -86,6 +86,7 @@ def __init__(
         timeout: float = 1.0,
         is_debug: bool = False,
         loop_mode: Optional[str] = None,
+        runtime: Optional[BaseRuntime] = None,
```

Comment:
why not `runtime = BaseRuntime`?

Reply:
So to maintain the consistency to current implementation?

Reply:
changed to `runtime_mode` similar to `loop_mode`